forge-ai/src/main/java/forge/ai/ComputerUtilCombat.java

```diff
@@ -223,7 +223,7 @@ public static int poisonIfUnblocked(final Card attacker, final Player attacked)
         int damage = attacker.getNetCombatDamage();
         int poison = 0;
         damage += predictPowerBonusOfAttacker(attacker, null, null, false);
-        if (attacker.hasKeyword(Keyword.INFECT)) {
+        if (attacker.hasKeyword(Keyword.INFECT) || attacker.hasKeyword(Keyword.TOXIC)) {
```

Comment:
this seems wrong, Toxic has its own magnitude
`predictPoisonFromTriggers` should already consider it

Reply:
Oh right. I'll rename that function since I didn't realize it already had something in it.